README.adoc

```diff
@@ -0,0 +1,69 @@
+== funops-cli
+
+A library that provides helpers and DRYer import possibilities
+and enhances the following battled tested libraries alread on
+Hackage for our system scripting work in a type safe environment:
+
+* `aeson`
+* `wreq`
+* `doctest`
+* `optparse-applicative`
+* `turtle`
+
+=== Purpose
+
+Provide an API that doesn't require the author of a system CLI script
+in Haskell to know details of `Control.Applicative`, `Control.Monad`,
+`Data.Maybe`, `Data.Aeson`, `Data.Lens`, combinators and where to
+import them from.
+
+=== Assumptions
+
+We work in the Nix environment. So we assume you have Nix package
+manager setup with GHC 7.10.x support.
+
+=== Scope
+
+This library will provide a simpler API for the purposes of writing
+system scripts for the common use cases of:
```

Comment:
Having said the above I still changed this part to be more specific about my intent here and the goals of the project.